smart-contracts/truffle-config.js

```diff
@@ -43,6 +47,18 @@ module.exports = {
     // options below to some value.
     //
     development: {
+      host: '127.0.0.1', // Localhost (default: none)
+      port: 8545, // Standard Ethereum port (default: none)
+      network_id: '1337', // Any network (default: none)
+    },
+
+    test: {
+      host: '127.0.0.1', // Localhost (default: none)
+      port: 7545, // Standard Ethereum port (default: none)
+      network_id: '5777', // Any network (default: none)
+    },
+
```

Comment:
They are 2 networks called "test" in truffle config, one with id '5777' and one with '*'

Reply:
ohh, one of them should not exist. Thanks :clap: